src/main.scss

```diff
@@ -6,12 +6,14 @@
 @include font-face("SourceSansPro", "assets/fonts/sourcesanspro-regular-webfont", normal, normal);
 @include font-face("SourceSansPro", "assets/fonts/sourcesanspro-semibold-webfont", 600, normal);
 
+@import "src/styles/font-styles";
+@import "src/styles/forms";
+@import "src/styles/buttons";
+
 body {
-    padding: 0;
-    font-family: $fonts;
-    font-size: $fsBase;
-}
+    @extend .text;
+    @extend .text--base;
 
-button {
-  font-family: inherit;
+    padding: 0;
```

Comment:
should be on in button component. same for color - below

Reply:
Oops, I thought I had already removed it ^^'